content/en/blog/2023/traffic-for-ambient-and-sidecar/index.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "Deep Dive into the Network Traffic Path of the Coexistence of Ambient and Sidecar"
+description: "Deep Dive into the Traffic Path of the Coexistence of Ambient and Sidecar."
+publishdate: 2023-09-15
+attribution: "Steve Zhang (Intel), John Howard (Google), Yuxing Zeng(Alibaba)"
+keywords: [traffic,ambient,sidecar,coexistence]
+---
+
+There are 2 modes for Istio: Ambient and Sidecar, the former is still on the way, the latter is the classic one. therefore, the coexistence of Ambient and Sidecar should be a normal deployment form. That's the reason why this blog may be helpful for Istio users.
+
+## Background
+
+In the architecture of modern microservices, communication and management among services is critical. To address the challenge, Istio emerge as a service mesh technology. It provides traffic control, security, and superior observation capabilities by importing the sidecar. In order to further improve the adaptability and flexibility of Istio, the Istio community began to explore a new evolution mode - Ambient mode. In this mode, Istio no longer relies on explicit sidecar injection, but achieve the communication and management among services through ztunnel and waypoint proxies. Ambient also brings a series of improvements, such as lower resource consumption, simpler deployment, more flexible configuration options and non-restarting for pod when enabling ambient, enabling Istio to play a better role in various scenarios.
```

Comment:
```suggestion
In modern microservices architecture, communication and management among services are critical. To address the challenge, Istio has emerged as a service mesh technology. It provides traffic control, security, and superior observation capabilities by introducing the sidecar. To further enhance the adaptability and flexibility of Istio, the Istio community has begun exploring a new dataplane mode - Ambient mode. In this mode, Istio no longer relies on explicit sidecar injection, but achieves the communication and management among services through ztunnel and waypoint proxies. Ambient also brings a series of improvements, such as lower resource consumption, simpler deployment, more flexible configuration options and eliminating needs for pod restarts when enabling ambient, enabling Istio to play a better role in various scenarios.
```

Reply:
I think there may be a capitalization issue of word `Ambient`, some are lower cases and some are upper cases.

Reply:
Yeah, thanks @hanxiaop, I'm checking on it ^_^

Reply:
`ambient` is lower-case. If I remember correctly, the only time it should be capitalized is when it's grammatically correct to do so (ie, headers and beginning of sentences).

Reply:
okay, got it, I will update all of them